test/toucan2/tools/after_update_test.clj

```diff
@@ -175,3 +177,27 @@
              (generated-name `(after-update/define-after-update :model-2
                                 [~'venue]
                                 ~'venue)))))))
+
+(derive ::people.bird-lovers ::test/people)
+
+;; Unfortunately the changes-available-test needs to do something side-effecty since the value of after-update is
+;; discarded. Toggling an atom seemed like the easiest side effect.
+(def ^:private bird-lover-found? (atom false))
+
+(after-update/define-after-update ::people.bird-lovers
+  [person]
+  (when ((fnil str/includes? "") (:name (protocols/changes person)) "Cam")
+    (reset! bird-lover-found? true))
+  person)
+
+(deftest ^:synchronized changes-available-test
+  (test/with-discarded-table-changes :people
+    (testing "Changes made via update should be available in after-update"
+      (let [[{row-id :id}] (insert/insert-returning-instances! ::people.bird-lovers
+                                                      {:name "Gwynydd Purves Wynne-Aubrey Meredith"})]
+        ;; Update without the relevant changes:
+        (update/update! ::people.bird-lovers row-id {:created-at (LocalDateTime/parse "2017-01-01T00:00")})
+        (is (false? @bird-lover-found?))
```

Comment:
it would be good to reset this to false at the beginning of the test so if you run the test more than once from a REPL it can still pass

Reply:
@camsaul good point; just fixed it